.github/ISSUE_TEMPLATE/migration.yml

```diff
@@ -0,0 +1,70 @@
+name: "\U0001F4DA Migration from pytorch-pretrained-bert or pytorch-transformers"
+description: Report a problem when migrating from pytorch-pretrained-bert or pytorch-transformers to transformers
+labels: [ "migration" ]
```

Comment:
Same as above regarding the label 'migration' appearing under 'migration from...' like 'bug' in 'bug report'.